tests/encryption_/tests.py

```diff
@@ -0,0 +1,47 @@
+from django.core import management
+from django.db import connection
+from django.test import TestCase, modify_settings
+
+from django_mongodb_backend import encryption
+
+from .models import Person
+
+
+class EncryptedModelTests(TestCase):
+    databases = {"default", "encrypted"}
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.person = Person(ssn="123-45-6789")
+        cls.person.save()
+
+    def test_encrypted_fields_map(self):
+        """ """
+        expected = {
+            "fields": [
+                {
+                    "path": "ssn",
+                    "bsonType": "string",
+                    "queries": [{"contention": 1, "queryType": "equality"}],
+                }
+            ]
+        }
+        with connection.schema_editor() as editor:
```

Comment:
Must use the encrypted connection (`connection` is the default):
```suggestion
        with connections["encrypted"].schema_editor() as editor:
```